theme/default/js/xibo-forms.js

```diff
@@ -37,16 +37,23 @@ var text_callback = function(dialog)
     // Conjure up a text editor
     CKEDITOR.replace("ta_text");
     
-    CKEDITOR.instances["ta_text"].on('instanceReady', function() {
+    CKEDITOR.instances["ta_text"].on('instanceReady', function(activeInstance) {
         var scale = $('#layout').attr('designer_scale');
 
-        $("#cke_ta_text .cke_contents").css({
-            background: $('#layout').css('background-color')
-        });
-        
-        $("#cke_ta_text iframe").css({
-            "background": "transparent"
+        // Set something sensible based on the color of the layout background and get the current background color
+        var color = $c.complement($("#layout").data().backgroundColor);
+        var bg = $("#layout").data().backgroundColor;
+
+        // Apply styles on the active editor instance
+        activeInstance.editor.document.getBody().setStyle('color', color);
```

Comment:
The intention was to do this on Line 14 - but it seems like I only took into account media with templates (like tickers). Plain old text modules don't have templates and therefore they won't be hit by the code on Line14.

I think that code is correct - but it needs to be changed to output the span if there isn't a template field present (we can tell by the override checkbox field being present)